sleuth/sleuth/enrich_tls.py

```diff
@@ -44,120 +44,58 @@
 VALID = 2
 
 
-security_categories = {
-    "unknown": 0,
-    "avoid": 1,
-    "legacy": 2,
-    "acceptable": 3,
-    "recommended": 4,
-}
+class Policy:
+    'Class to import seclevel policy'
```

Comment:
The usual convention for Docstrings in python is to use " or """ """ (double quotes)